app/src/main/java/com/lastaoutdoor/lasta/data/db/ActivitiesRepositoryImpl.kt

```diff
@@ -0,0 +1,70 @@
+package com.lastaoutdoor.lasta.data.db
+
+import com.google.firebase.auth.FirebaseUser
+import com.google.firebase.firestore.FieldValue
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.SetOptions
+import com.lastaoutdoor.lasta.data.model.profile.ActivitiesDatabaseType
+import com.lastaoutdoor.lasta.repository.ActivitiesRepository
+import javax.inject.Inject
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.tasks.await
+
+private const val ACTIVITIES_DB_NAME = "activities_database"
```

Comment:
Should have this in string resources

Reply:
Thank you for the feedback. I've now changed it!